custom_images/redis-3.2/sentinel/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie-slim
+
+ADD ["https://github.com/18f/dig-a/releases/download/v1.0.1/dig-a_linux_amd64", "/dig-a"]
+ADD ["https://github.com/18f/dig-srv/releases/download/v1.0.0/dig-srv_linux_amd64", "/dig-srv"]
```

Comment:
Hey @pburkholder, yeah. @LinuxBozo found that there were a bunch of dependencies being pulled in to install `dig` since you have to install `dnsutils`. It's over 60MB compared to leveraging the Golang standard library and building it into a 2.41 MB binary to get the same information.

I'm not sure that the `30` in the stock `dig` output is important here. [It's the TTL for the record](https://en.wikipedia.org/wiki/SRV_record#Record_format). In this case 30 seconds. I see the value hardcoded in [the `dig-srv` repo](https://github.com/18F/dig-srv/blob/master/digsrv.go#L24-L26), probably because it's not something that the `net` library returns in [the `SRV` struct](https://golang.org/src/net/dnsclient.go?s=5385:5472#L185).

Taking a look at the difference in output with `dig-a`, I'm seeing the same issue with the TTL being set to zero, but we still get the dependency size savings of 2.4 MB for that release too. The output looks the same except that the name doesn't include a period at the end of the domain using `dig-a` versus `dig a +nocmd [...]`.

<details>
<summary><code>root@redis-sentinel-0:/# apt-get install dnsutils</code></summary>

```shell
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following extra packages will be installed:
  bind9-host geoip-database ifupdown iproute2 isc-dhcp-client isc-dhcp-common krb5-locales libalgorithm-c3-perl libarchive-extract-perl libatm1 libbind9-90
  libcgi-fast-perl libcgi-pm-perl libclass-c3-perl libclass-c3-xs-perl libcpan-meta-perl libdata-optlist-perl libdata-section-perl libdns-export100
  libdns100 libfcgi-perl libgdbm3 libgeoip1 libgssapi-krb5-2 libirs-export91 libisc-export95 libisc95 libisccc90 libisccfg-export90 libisccfg90 libk5crypto3
  libkeyutils1 libkrb5-3 libkrb5support0 liblog-message-perl liblog-message-simple-perl liblwres90 libmodule-build-perl libmodule-pluggable-perl
  libmodule-signature-perl libmro-compat-perl libpackage-constants-perl libparams-util-perl libpod-latex-perl libpod-readme-perl libregexp-common-perl
  libsoftware-license-perl libssl1.0.0 libsub-exporter-perl libsub-install-perl libterm-ui-perl libtext-soundex-perl libtext-template-perl libxml2
  libxtables10 netbase perl perl-base perl-modules rename sgml-base xml-core
Suggested packages:
  rblcheck ppp rdnssd net-tools iproute2-doc resolvconf avahi-autoipd geoip-bin krb5-doc krb5-user perl-doc libterm-readline-gnu-perl
  libterm-readline-perl-perl make libb-lint-perl libcpanplus-dist-build-perl libcpanplus-perl libfile-checktree-perl libobject-accessor-perl sgml-base-doc
  debhelper
Recommended packages:
  libarchive-tar-perl
The following NEW packages will be installed:
  bind9-host dnsutils geoip-database ifupdown iproute2 isc-dhcp-client isc-dhcp-common krb5-locales libalgorithm-c3-perl libarchive-extract-perl libatm1
  libbind9-90 libcgi-fast-perl libcgi-pm-perl libclass-c3-perl libclass-c3-xs-perl libcpan-meta-perl libdata-optlist-perl libdata-section-perl
  libdns-export100 libdns100 libfcgi-perl libgdbm3 libgeoip1 libgssapi-krb5-2 libirs-export91 libisc-export95 libisc95 libisccc90 libisccfg-export90
  libisccfg90 libk5crypto3 libkeyutils1 libkrb5-3 libkrb5support0 liblog-message-perl liblog-message-simple-perl liblwres90 libmodule-build-perl
  libmodule-pluggable-perl libmodule-signature-perl libmro-compat-perl libpackage-constants-perl libparams-util-perl libpod-latex-perl libpod-readme-perl
  libregexp-common-perl libsoftware-license-perl libssl1.0.0 libsub-exporter-perl libsub-install-perl libterm-ui-perl libtext-soundex-perl
  libtext-template-perl libxml2 libxtables10 netbase perl perl-modules rename sgml-base xml-core
The following packages will be upgraded:
  perl-base
1 upgraded, 62 newly installed, 0 to remove and 7 not upgraded.
Need to get 17.6 MB of archives.
After this operation, 60.7 MB of additional disk space will be used.
Do you want to continue? [Y/n]
```
</details>